test/citestutils.js

```diff
@@ -282,12 +296,12 @@ function testRestore(params, inputStream, databaseName, callback) {
   promises.unshift(pipeline(pipelineStreams));
 
   // Wait for the all the promises to settle and then assert based on the process promise
-  return Promise.allSettled(promises)
+  return await Promise.allSettled(promises)
```

Comment:
as for backup case there shouldn't be a need to `await` here
```suggestion
  return Promise.allSettled(promises)
```